db/seeds.rb

```diff
@@ -1,34 +1,37 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+# frozen_string_literal: true
+
+# This file should contain all the record creation needed to seed the database
+# with its default values. The data can then be loaded with the rake db:seed
+# (or created alongside the db with db:setup).
 require 'yaml'
 
 # We need mail_configuration to create a user account, because it sends a confirmation email.
 MailConfiguration.apply!
 
-puts "[db:seed] Seeding Roles"
-Spree::Role.where(:name => "admin").first_or_create
-Spree::Role.where(:name => "user").first_or_create
+Rails.logger.debug "[db:seed] Seeding Roles"
```

Comment:
I'm not sure, but it might be useful to output some of this info as `info` (so that it appears on server environments for example)

Reply:
Okay, happy to change to info. I guess that I don't look at that anymore and don't need to see it.

Reply:
Thanks, I thought it might be helpful to confirm what changes have been made in the DB.